### ISSUE TITLE
Always use the latest Raspberry Pi firmware

### ISSUE THREAD

**Issue by swvanbuuren** [swvanbuuren/debos-rpi4#1]
https://github.com/swvanbuuren/debos-rpi4/blob/9e1dc6021e21159b85df088bdb75c6947e33dea1/base-pack.yaml#L17

Proposal to replace the URL with the following command:

```console
curl -s https://api.github.com/repos/raspberrypi/firmware/releases | grep browser_download_url | head -n 1 | cut -d '"' -f 4
```

This assures the build always uses the latest firmware. Also, it avoids errors, when older releases are removed.

**Comment by swvanbuuren** [comment]
We actually need a different file.

The following retrieves the correct URL:
```bash
http://github.com/raspberrypi/firmware/archive/refs/tags/$(curl -s https://api.github.com/repos/raspberrypi/firmware/tags | grep name | head -n 1 | cut -d '"' -f 4).tar.gz
```

**Comment by swvanbuuren** [comment]
Fixed with #2
